content/influxdb/cloud/account-management/pricing-plans.md

```diff
@@ -58,7 +58,7 @@ The following vectors determine pricing:
 - **Data out** is the total sum of the data (measured in GB) returned to the user to answer a query, also known as data transfer costs.
 - **Query count** is the total number of individual query operations:
```

Comment:
I don't think so. Delete requests are not a pricing vector.

Reply:
Thanks, @philjb --I see what you mean for pricing vectors...think I meant to ask this question for "data usage" on this page: https://docs.influxdata.com/influxdb/cloud/account-management/data-usage/ cc @bpalani 

Specifically, when folks exceed the specified number of delete requests within 5 minutes:
 - Do we show them an indicator on the Usage page...screenshot currently shows drop-down for reads/writes requests over the past x amount of time. 
 - Can folks raise their delete request limits? If not, maybe to say to raise write and read limits, rather than raise rate limits, as we've grouped deletes in here.
 - Do we tell folks what the "specified" number of delete requests per 5 minutes is?